src/chromium/crCoverage.ts

```diff
@@ -82,7 +99,7 @@ class JSCoverage {
     this._resetOnNavigation = resetOnNavigation;
     this._reportAnonymousScripts = reportAnonymousScripts;
     this._enabled = true;
-    this._scriptURLs.clear();
+    this._scriptIds.clear();
     this._scriptSources.clear();
     this._eventListeners = [
       helper.addEventListener(this._client, 'Debugger.scriptParsed', this._onScriptParsed.bind(this)),
```

Comment:
Thanks for this! I've tested a simple hack with raw coverage and it works nicely, apart from:

5 lines down from this we have:
```
this._client.send('Profiler.startPreciseCoverage', {callCount: false, detailed: true}),
```
I can get a really nice coverage report with istanbul if we get callCount info:
```
this._client.send('Profiler.startPreciseCoverage', {callCount: true, detailed: true}),
```

Reply:
Sounds good, counting calls now.